src/backend/alembic/versions/5c3adc3d6a0a_.py

```diff
@@ -0,0 +1,30 @@
+"""empty message
+
+Revision ID: 5c3adc3d6a0a
+Revises: ed17f144f4bf
+Create Date: 2024-07-25 16:13:07.522825
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "5c3adc3d6a0a"
+down_revision: Union[str, None] = "ed17f144f4bf"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint(None, "conversations", ["id"])
```

Comment:
Is there a use for this unique constraint on the `id` column since this is the PrimaryKey? From what I understand, it is unique by default

Reply:
Couple things I noticed:

- We override the base definition here for conversations https://github.com/cohere-ai/cohere-toolkit/blob/main/src/backend/database_models/conversation.py#L15, so it is not unique. The PK for conversations is id+user_id, so I suppose that was why.
- If we do not add unique constraint here we can't create a relation between convosation and conversationfileassociation. ie. ConversationFileAssociation.conversation and Conversation.conversation_file_associations